modules/ROOT/pages/access-logging.adoc

```diff
@@ -72,17 +70,17 @@ The following table lists the available log format options.
 |The local IP address
 
 |%b
-|The response size in bytes, excluding the headers
+|The response size in bytes excluding the headers
 
 |%B
-|The response size in bytes, excluding the headers
-If no value is found, `0` is printed instead of `-`.
+|The response size in bytes excluding the headers
```

Comment:
add a period to this one, even though it's not a complete sentence, to separate it from the complete sentence that follows